ui/raidboss/data/03-hw/raid/a8s.js

```diff
@@ -521,6 +540,7 @@
       alertText: {
         en: 'Min HP: Provoke Boss => Late NE Tornado',
         de: 'Min HP: Boss herrausfordern => Später No Tornado',
+        fr: 'PV Min : Provocation Boss => Cyclone NO ou NE en retard',
```

Comment:
Can you keep this consistent with the English version? I don't think it's good to have different translations suggest different things.  I think that means that the German version is also inconsistent.  cc @Akurosia

Reply:
As mentioned above, it pops in NW or NE. I wanted to do this because if the indication is to go NW, but the tornado is in NE, it is no longer consistent, right?

Reply:
Here are a few thoughts:
* Overall, I think the most important thing is to have the same callout in all languages.  I don't want to tell English players to run left and then German players to run right and French players to run right or left as they feel like.  Ideally, translations to different languages should tell the players to do exactly the same thing.
* For this mechanic, it is not "one tornado pops in NW or NE", it is "four tornados appear in the corners and you need to pick which players go in which one".  You could theoretically send any of the four people taking tornados to any of the four tornados.  You could send different people to different orbs too, although that'd be more running.  There's not one way to solve this mechanic.  However, most players seem to consistently solve this by sending particular players north for tornados, although there's some disagreement on which north tornado gets taken.
* In most cases, per [this doc](https://github.com/quisquous/cactbot/blob/master/CONTRIBUTING.md#trigger-text) it says "Don't prescribe a particular strategy, if multiple strategies exist, i.e. Titania Ex tethers or Hello World".  Subjectively, I think in this case, this content is old, so it's more ok to pick a strategy and tell people what to do.  I picked the Death and Taxes guide linked in the timeline: https://dtguilds.enjin.com/a8sguide.  This is also opposite the https://www.bluegartr.com/threads/128379-Alexander-Midas-IV-(Savage)-AS8-Guide which says to do the other north tornado.
* Similar to my comments above, I think this is a case of "why are you changing this as a part of French translation".  If your concern was "let's change the a8s strategy" then do that separately and not as a translation pull request.

Those are all the thoughts that I have going on.

I think there are two options to resolve this:
1) pick a strategy, and tell players to go to a single tornado.  I think they'll figure it out if somebody takes the other north tornado.  I (slightly) think it'd be better to stick with the a8s guide linked in the timeline and tell Min HP debuff players to take the NE tornado.
2) tell players to just take a "north" tornado, and change the other texts to be the same.  I think this is slightly worse and more confusing, but if other people think this is better, I think it's workable.

Reply:
As I do the unsync content, and that on the videos the strats change sometimes. I ignored the basic strategy on HW. You have convinced me. I prefer to restore and translate simply.
Sorry to have taken your time at think.